paper/paper.md

```diff
@@ -0,0 +1,233 @@
+```
+
+We apply this code to data generated by a Ricker model presenting an abrupt transition
+at $t = 860$, which is used in the first tutorial of `ewstools` [@bury_ewstools_2023],
+the most recent software with functionalities that are similar to TransitionsInTimeseries.jl.
+The results are shown in [Fig. 1](@figure1) and display, as expected from CSD theory, an
+increase in both variance and AR1 coefficient, which is exactly the same as computed
+by `ewstools`. However, calling `signif.pvalues` shows that the increase in variance is not
+significant ($p = 0.284$), whereas the increase in AR1 coefficient is ($p = 0.001$).
+
+![Output of plotting function in usage example.\label{fig: fig1}](figures/figure1.png)
+
+We believe that a concise and unambiguous code will greatly reduce the programming effort of
+many researchers and ease the code reviewing process. Finally, the code documentation
+provides additional examples, showcasing that the simplicity of the code also applies to
+real-world applications.
+
+## Performance
+
+TransitionsInTimeseries.jl is written in Julia, which offers both a simple syntax and good
+performance. Additionally, the run-time of all performance-relevant steps have been
+optimized. Significance testing can be the main bottleneck and was therefore parallelised, 
+which simply requires the user to activate more threads in their Julia REPL. This is of
+particular importance since the transition analysis is sometimes performed on 10,000+
+timeseries [@ben-yami_uncertainties_2023], which can last hours with a serial code that is
+not optimized.
+
+## Reproducibility
+
+Some steps of a transition analysis involve random number generators, which need 
+to be handled with care in parallelized codes. This is done in TransitionsInTimeseries.jl, 
+which offers the possibility of seeding a random number generator by using the 
+keywordargument `rng`, as done in the example shown above. Furthermore, 
+TransitionsInTimeseries.jl follows the guidelines of semantic versionning which, 
+along with Julia's integrated package manager, ensures that the same code is used for both 
+results generation and peer reviewing.
+
+## Reliability
+
+In high-impact context mentioned above, errors are to be avoided with particular care.
+TransitionsInTimeseries.jl is therefore tested via continuous integration on a large test
+suite, thus providing a reliable research framework. Furthermore, a centralised code base
+implies that any new user is a new test, thus increasing the reliability of the code over
+time. Finally, the robustness of the results with respect to a parameter, e.g. the width
+of the sliding window, can be easily studied thanks to the simple syntax, thus contributing
+to the reliability of the results.
+
+## Versatility
+
+TransitionsInTimeseries.jl detaches the degrees of freedom that are available to the user
+from the analysis pipeline. Methods for detection and prediction can thus be applied equally
```

Comment:
I agree but I struggle in explaining it in a concise way. We are already beyond the max number of words...

Reply:
We should worry about the limit after submission. If they complain we can cut out the least important sections completely.

But for this particular point I think we can just say "can be used for either hindcast or forecast " , which, I personally do not find noteworthy as anything like that could be used for either as it's a matter of perspective.

Reply:
Ok for word limit!

Hindcast and detection are not the same. Whereas people doing hindcasting obviously need to detect a transition, some might just want to do the detection without the hindcasting. I personally find the formulation of the current version understandable, since prediction and detection are better introduced from the beginning